resources/views/admin/userview.blade.php

```diff
@@ -0,0 +1,86 @@
+@extends('layouts.app')
```

Comment:
Ik zie dat je _@extends_ methode gebruikt. Ik heb zelf de _<x- component_ layout gebruikt bij mijn code. We moeten even kijken of we allebei de manieren kunnen blijven gebruiken of dat we er eentje moeten kiezen omdat het in conflict komt met de gehele code.